dom0-updates/qfile-dom0-unpacker.c

```diff
@@ -49,6 +49,7 @@ int main(int argc, char ** argv)
 {
 	const char *incoming_dir;
 	int uid;
+    int i;
```

Comment:
Unintended tab?

Reply:
The other way around - the code is formatted with tabs, which I'm going to fix in separate PR.